oam/install_scripts/columnstore-post-install.in

```diff
@@ -81,6 +81,12 @@ if [ -z "$(pgrep -x mariadbd)" ];then
 fi
 sleep 2
 
+# One time remove x-columnstore.cnf if it exists.
+# x-columnstore.cnf was a kludge for using when plugin-maturity=gamma
+if [ -f @MARIADB_MYCNFDIR@/x-columnstore.cnf]; then
```

Comment:
@abarkov suggests a better approach to handle the file. It is more wise to rename it b/c it could potentially contain user-made configurations.

Reply:
e.g. to x-columnstore.cnf.backup